### PR TITLE
Adding business descriptions for Knot.x.

### PR DESCRIPTION
**Pull request by tomaszmichalak**
READY.
---

I hereby agree to the terms of the Knot.x Contributor License Agreement.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,16 +3,81 @@
...
+Find out more about this topic reading [Knot.x Demo](https://github.com/Cognifide/knotx/wiki/RunningTheDemo) 
+section.
+
+###Extensions
+You need to implement custom authentication mechanism for your site and then integrate with service 
+talking with its own custom protocol. **Knot.x** is fully modular platform with very flexible extension
+points: [Knots](https://github.com/Cognifide/knotx/wiki/Knot) and [Adapters](https://github.com/Cognifide/knotx/wiki/Adapter).
+Those extension points communicates with Knot.x Core using very performant Event Bus so you can
+implement your integration layer in one place inside Knot.x. Not enough? If you wish you can implement
+your extensions in language you like thanks [Vert.x](http://vertx.io/) capabilities.
```
Comment (Skejven) [inline README.md]: > If you wish you can implement your extensions in language you like thanks to [Vert.x polyglot](http://vertx.io/) feature.

File: README.md [hunk 2]
```diff
@@ -3,16 +3,82 @@
...
+## What problems does Knot.x solve?
+
+###Features
+
+<img align="right" 
+  src="https://github.com/Cognifide/knotx/blob/master/documentation/src/main/wiki/assets/knotx-intro-features.png?raw=true"
+  alt="Features"/>
+
+Probably you have many features / services you want to connect to your site. They come from 
+different vendors, talk using various protocols (REST / SOAP / binary) and have different
```
Comment (marcinczeczko) [inline README.md]: 'talk using various protocols (REST / SOAP / binary)'
into
'talk using various network protocols (HTTP/WebSocket/TCP etc.) and with message format (SOAP/JSON/XML/binary etc.)'

File: README.md [hunk 3]
```diff
@@ -57,7 +111,7 @@ For bugs, questions and discussions please use the [Github Issues](https://githu
 
 ## Demo
 
-You can run **Knot.x** demo with less than 5 minutes, you only need Java 8. See [how to run Knot.x demo](https://github.com/Cognifide/knotx/wiki/RunningTheDemo).
+You can run a **Knot.x** demo within less than 5 minutes, all you need is Java 8. See [how you can run the Knot.x demo](https://github.com/Cognifide/knotx/wiki/RunningTheDemo).
```
Comment (marcinczeczko) [inline README.md]: you also need maven :)

Comment (tomaszmichalak) [inline README.md]: only run - java

**Pull request merged**